dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets

```diff
@@ -8,13 +8,23 @@
   <!-- It should not display any schema warnings in error list window.            -->
   <!-- ========================================================================== -->
 
-  <!-- With .NET 5, the propery TargetPlatformIdentifier is not UAP but rather windows. This file expects UAP for all .NET versions, so adjust. -->
+  <!-- With .NET 5, the propery TargetPlatformIdentifier is not UAP but rather windows.
+       Likewise, for other console apps, TargetPlatformIdentifier is windows.
```

Comment:
>windows [](start = 69, length = 7)

Exe?

Reply:
TargetPlatformIdentifier is windows for console apps, I think. It's certainly not UAP. I'll confirm later that it's windows.

I know we enter the else clause below, that's how I know it's not UAP, and why I suspect it's Windows (DesktopFileName is Windows.props, whereas UapFileName is UAP.props).

S:\vs\src\vsproject\PackageAndDeploy\Shared.Packaging\WindowsPropsLoader.cs:
```
            if (TargetPlatformIdentifier.IsPhone(targetPlatformIdentifier))
            {
                return PhoneFileName;
            }
            else if ("UAP".Equals(targetPlatformIdentifier, StringComparison.OrdinalIgnoreCase))
            {
                return UapFileName;
            }
            else
            {
                return DesktopFileName;
            }
```

Reply:
this conflicts with line 18-19.

Reply:
@huichen123 OutputType of Exe corresponds to console apps. We need to set TargetPlatformIdentifier for console apps to UAP. That's what we're doing. I don't see an issue. Can you elaborate?

Reply:
your previous comment said "TargetPlatformIdentifier is **windows** for console apps".